core/src/tile/tileManager.cpp

```diff
@@ -97,61 +104,60 @@ std::shared_ptr<ClientGeoJsonSource> TileManager::getClientSourceById(int32_t _i
     return nullptr;
 }
 
-void TileManager::tileProcessed(std::shared_ptr<TileTask>&& task) {
+void TileManager::tileProcessed(std::shared_ptr<TileTask>&& _task) {
     std::lock_guard<std::mutex> lock(m_readyTileMutex);
-    m_readyTiles.push_back(std::move(task));
+    m_readyTiles.push_back(std::move(_task));
 }
 
-bool TileManager::setTileState(Tile& tile, TileState state) {
+bool TileManager::setTileState(Tile& _tile, TileState _newState) {
```

Comment:
We should have unit tests for this, will help in any future modifications and will catch issues we catch the hard way in this branch.